scripts/statistics.js

```diff
@@ -23,6 +23,12 @@ const { argv } = require('yargs').command(
         describe: 'Show statistics for all browsers within BCD',
         type: 'boolean',
         nargs: 0,
+      })
+      .option('asCount', {
```

Comment:
Not opposed to this naming (being consistent with the function parameters below), but wondering whether an all-lowercase option would be more "conventional" (don't recall seeing mixed case options in the wild):

```suggestion
      .option('counts', {
```